quantum/secure.h

```diff
@@ -65,3 +65,15 @@ void secure_keypress_event(uint8_t row, uint8_t col);
 /** \brief Handle various secure subsystem background tasks
  */
 void secure_task(void);
+
+/** \brief quantum hook called when changing secure status device
+ */
+void secure_hook_quantum(secure_status_t secure_status);
```

Comment:
My concern here is adding of additional hooks to the file.

Would it be better to leave the kb/user functions here, and move the quantum hook into quantum.c, so that other things can be added there?